src/ol/renderer/vector.js

```diff
@@ -275,6 +276,32 @@ function renderLineStringGeometry(
   }
 }
 
+/**
+ * @param {import("../render/canvas/BuilderGroup.js").default} builderGroup Replay group.
+ * @param {import("../geom/CircularString.js").default|import("../render/Feature.js").default} geometry Geometry.
+ * @param {import("../style/Style.js").default} style Style.
+ * @param {import("../Feature.js").FeatureLike} feature Feature.
+ * @param {import("../render/canvas/BuilderGroup.js").default} [opt_declutterBuilderGroup] Builder for decluttering.
+ */
+function renderCircularStringGeometry(
+  builderGroup,
+  geometry,
+  style,
+  feature,
+  opt_declutterBuilderGroup
+) {
+  const strokeStyle = style.getStroke();
+  if (strokeStyle) {
+    const circularStringReplay = builderGroup.getBuilder(
+      style.getZIndex(),
+      BuilderType.CIRCULAR_STRING
+    );
+
+    circularStringReplay.setFillStrokeStyle(null, strokeStyle);
+    circularStringReplay.drawCircularString(geometry, feature);
+  }
```

Comment:
Bij andere objecten wordt ook nog iets met tekst (labels) gedaan. Geen idee hoe je dat moet forceren in OpenLayers, misschien een label toevoegen aan een Feature?

Voor OpenLayers acceptatie moet dit er vast nog wel bij

Reply:
Ik verwacht ook dat dit een ding zal zijn. Wilde het alleen voor nu toch even negeren. Als het hier over gaat ben ik hoopvol genoeg dat men in ieder geval in de rest geïnteresseerd is.